ssz-rs/src/bitvector.rs

```diff
@@ -95,7 +93,9 @@ impl<const N: usize> Bitvector<N> {
 
     fn pack_bits(&self) -> Result<Vec<u8>, MerkleizationError> {
         let mut data = vec![];
-        let _ = self.serialize(&mut data)?;
+        let _ = self
+            .serialize(&mut data)
+            .map_err(|_| MerkleizationError::SerializationError);
```

Comment:
same thing here?

Reply:
Addressed in https://github.com/ralexstokes/ssz-rs/pull/25/commits/b58448c30caa3cfc15bf696c020abb01001cac89.